src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/VetFieldEnum.java

```diff
@@ -121,7 +121,26 @@ public String getColumnValue(final VariantContext variant) {
 
     AS_RAW_MQRankSum { // TODO -- maybe rely on 1/1 for call_GT, also get rid of the | at the beginning
         public String getColumnValue(final VariantContext variant) {
+            // e.g. AS_RAW_MQRankSum=|1.4,1|NaN;
             String out =  getAttribute(variant, GATKVCFConstants.AS_RAW_MAP_QUAL_RANK_SUM_KEY, null);
+
+            if (out == null) {
+                // Try to use non-AS version
+                // TODO: it looks like the AS_RAW version also trims to a single decimal point??
+                // e.g. MQRankSum=1.465 and turn it into |-0.511,1|
```

Comment:
I'm confused here, how did 1.465 become -0.511?

Reply:
lol -- cut and paste!